src/app/branded/step-1/branded-checkout-step-1.tpl.html

```diff
@@ -73,7 +73,7 @@ <h3 class="panel-name" translate>{{'CHOOSE_RESOURCE'}}</h3>
           </div>
         </div>
         <div class="col-sm-4">
-          <img ng-src="{{$ctrl.premiumImageUrl}}" style="width: 200px; height: 200px" />
+          <img ng-src="{{$ctrl.premiumImageUrl}}" style="width: 200px" />
```

Comment:
It's probably better to make a class so that ministries can override this style.